package.json

```diff
@@ -88,7 +88,7 @@
     "identity-obj-proxy": "^3.0.0",
     "ip": "^1.1.5",
     "jest": "^21.0.2",
-    "lerna": "^2.1.2",
+    "lerna": "^2.5.1",
```

Comment:
Should we update `lerna` and `shelljs` in this PR?

Reply:
set them back to original